dietpi/dietpi-software

```diff
@@ -1199,6 +1199,15 @@ INDEX_BROWSER_TARGET=$INDEX_BROWSER_TARGET"
 		aSOFTWARE_CATX[$software_id]=8
 		aSOFTWARE_DOCS[$software_id]='https://dietpi.com/docs/software/cloud/#k3s'
 
+		#------------------
+		software_id=200
+
+		aSOFTWARE_NAME[$software_id]='DietPi Dashboard (beta)'
+		aSOFTWARE_DESC[$software_id]='Official DietPi web dashboard'
+		aSOFTWARE_CATX[$software_id]=8
+		aSOFTWARE_DOCS[$software_id]='https://dietpi.com/docs/dietpi_tools/#dietpi-dashboard'
+		aSOFTWARE_INTERACTIVE[$software_id]=1
```

Comment:
Shall we allow non-interactive installs which then default to `Stable`? Also we could add a `dietpi.txt` setting for this.

Reply:
Sounds good. Would we do this with the `$G_INTERACTIVE` variable?

Reply:
Our G_WHIPs are cancelled automatically in this case. We could initiate the variable with the setting in `dietpi.txt`, if it is empty assign `Stable`, then show the whiptail and if it is returns a selection override the variable again. So priority is: Whiptail selection > `dietpi.txt` > `Stable`